dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype-mbeans.properties

```diff
@@ -62,9 +59,6 @@ plugin.wsServerIOTimeout.desc=Identifies the maximum amount of time that the web
 plugin.wsServerIdleTimeout.name=Websocket idle connection timeout
```

Comment:
I know this is not part of the update here, but should `Websocket` be `WebSocket` ? Otherwise I dont think there's anything to review here for ID so just commenting as not to block.